client/src/components/Discussion.js

```diff
@@ -74,8 +74,8 @@ class Discussion extends Component {
     return (
       <div className="discussion">
         <div className="discussion-header">COMMENTS</div>
-        {this.renderMainPostBox()}
-        {this.renderComments()}
+        <div className="comment-box">{this.renderComments()}</div>
+        <div>{this.renderMainPostBox()}</div>
```

Comment:
This flips the comment entry from above the already posted comments to the below.  If there is a big discussion the user would have to scroll down all the way to comment. 
This doesn't have to be changed back but I wanted to discuss the user flow change.

Is this a change we want?

Reply:
Will defer to the group here... I can make a case for both. We should also consider rendering the comments from more recent > less recent instead of vice verse (how it is now)

Reply:
lets bring this up tonight at the meetup.